tests/e2e/sample-applications/mongo-persistent/mongo-persistent-block.yaml

```diff
@@ -89,13 +89,13 @@ items:
                   else
                     echo "Formatting $DEVICE"
                     mkfs.ext4 $DEVICE
+                    mkdir -p $MOUNT_POINT
+                    echo "Mounting $DEVICE in the $MOUNT_POINT"
+                    mount $DEVICE $MOUNT_POINT
+                    echo $(date +%s) > $MOUNT_POINT/format_timestamp
```

Comment:
That was kind of intentional, so if the filesystem exists on a block device it will be also mounted/unmounted to check if it's valid.

One change which could help here is to have `>>` instead of `>` in the `echo $(date +&s)` line so the data is appended to the file if it was exisiting.

Reply:
can this PR be closed?

Reply:
I would vote for it, unless @amastbau has strong feeling of reworking it. Note my comment that some logic was intentional here.